nobl9/resource_direct.go

```diff
@@ -792,6 +792,69 @@ func (s lightstepDirectSpec) UnmarshalSpec(d *schema.ResourceData, spec v1alphaD
 	return
 }
 
+// Logic Monitor Direct
+// https://docs.nobl9.com/Sources/logic-monitor#logic-monitor-direct
+const logicMonitorDirectType = "logic_monitor"
+
+type logicMonitorDirectSpec struct{}
+
+func (s logicMonitorDirectSpec) GetSchema() map[string]*schema.Schema {
+	logicMonitorSchema := map[string]*schema.Schema{
+		"account": {
+			Type:        schema.TypeString,
+			Description: "[required] | Account name.",
```

Comment:
It should be listed under `### Required` section once you generate the docs, no need to add this

```suggestion
			Description: "Account name.",
```